index.html

```diff
@@ -8,81 +8,90 @@
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">קשרי משקיעים</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">התחנות שלנו</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">פזומט</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">חברות הקבוצה</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">אחריות תאגידית</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">אודות</a>
+				        </li>
+		    		</ul>
+	    		</nav>
+    		</div>
+    	</div>
+    	<!-- HEADER 2 -->
 
-				<div class="low-sec-nav container hidden-md-down">	
-		    		<nav class="nav-1 navbar navbar-nav navbar navbar-expand-lg">
-			    		<ul class="d-flex flex-row">
-			    			<li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">צרו קשר</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">קריירה</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">קשרי משקיעים</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">התחנות שלנו</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">פזומט</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">חברות הקבוצה</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">אחריות תאגידית
-					        	</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">אודות
-					        	</a>
-					        </li>
-			    		</ul>
-		    		</nav>
+	    <!-- reg and personal erea -->
+		<div class="nav-2 container-fluid d-flex align-items-center justify-content-between">
+    		<div class="container d-flex justify-content-lg-between">
+    			<div class="l-side-gray-nav d-flex justify-content-lg-between col">
```

Comment:
Notice that the `col` class gives padding-left/right of `15px` which breaks your layout. You should reset the padding on `col`